proposed/links.md

```diff
@@ -87,23 +124,10 @@ interface LinkInterface
     /**
      * Returns a list of attributes that describe the target URI.
      *
-     * The list should be specified as a key-value list.
-     *
-     * There is no formal registry of the values that are allowed here, and
-     * validity of values is dependant on context.
-     *
-     * Common values are 'hreflang', 'title', and 'type'. Implementors
-     * embedding a serialized version of a link are responsible for only
-     * encoding the values they support.
-     *
-     * Any value that appears that is not valid in the context in which it is
-     * used should be ignored.
-     *
-     * Some attributes, (commonly hreflang) may appear more than once in their
-     * context. Attributes such as those may be specified as an array of
-     * strings.
-     *
-     * @return string[]
+     * @return array
```

Comment:
@michaelcullum that's the way @Crell and Drupal write inline descriptions. It's allowed by phpDoc.